CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ the detailed section referring to by linking pull requests or issues.
 #### Minor
 
 - Implemented markdown for asset descriptions
```

Comment:
UX: if the row is clickable, it's good that we added the cursor: pointer, but we should also highlight the row then, too